### PR TITLE
LPS-148307 Update maven project templates to use yarn as node manager

### PR DESCRIPTION
**Pull request by Seiphon**

### CODE REVIEW

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:relevant

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:project-templates

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:project-templates

**Comment by Seiphon** [comment]
ci:test:relevant

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:project-templates

**Comment by Seiphon** [comment]
ci:test:relevant

File: modules/sdk/project-templates/project-templates-npm-react-portlet/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -139,9 +139,6 @@
 						<goals>
 							<goal>yarn</goal>
 						</goals>
-						<configuration>
-							<arguments>install</arguments>
-						</configuration>
```
Comment (lawrence-lee) [inline modules/sdk/project-templates/project-templates-npm-react-portlet/src/main/resources/archetype-resources/pom.xml]: @Seiphon , do we need to also remove this configuration from vuejs?

Comment (Seiphon) [reply modules/sdk/project-templates/project-templates-npm-react-portlet/src/main/resources/archetype-resources/pom.xml]: I didn't add it for vue

**Comment by Seiphon** [comment]
resent here https://github.com/lawrence-lee/liferay-portal/pull/275

**Pull request closed without merging**